src/webapp/client/js/index.js

```diff
@@ -22,13 +22,13 @@ Currently, a globally scoped variable is used to store login information.
  client cookies.
 */
 var dbInfo = {
-	"host":null, "port":null, "database":null, "user":null, "password":null,
+	"host":'localhost', "port":54312, "database":null, "user":null, "password":null,
```

Comment:
I think this should stay null, shouldn't commit user defined data.

Reply:
I agree. The values should stay as null.